src/manage_nsfs/nsfs_schema_utils.js

```diff
@@ -32,19 +32,59 @@ schema_utils.strictify(account_schema, {
 // NOTE - DO NOT strictify nsfs_config_schema
 // we might want to use it in the future for adding additional properties
 
+/**
+ * validate_bucket_schema validates a account object against the NC NSFS account schema
+ * @param {object} account
+ */
 function validate_account_schema(account) {
     const valid = ajv.validate(account_schema, account);
-    if (!valid) throw new RpcError('INVALID_SCHEMA', ajv.errors[0]?.message);
+    if (!valid) {
```

Comment:
Please check `ajv.errors[0]` also there is no point in calling method without valid `ajv.errors[0]`, then we can remove the same check in multiple places in `create_schema_err_msg()`,